### DISCUSSION TITLE
'next/link' There is no splicing prefix for a label after packaging

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#31990] [Help]
### What version of Next.js are you using?

12.0.2

### What version of Node.js are you using?

12.0.2

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

 next build

### Describe the Bug

```
 <Link href="/about">
          <a>About Us</a>
        </Link>

```
 There is no splicing prefix for a label after packaging ， 

Online environment   Should be such an address
```
<a class="" target="_blank" rel="noreferrer" href="https://pctest.vrtbbs.com/about">About Us</a>

But for now:  <a class="" target="_blank" rel="noreferrer" href="/about">About Us</a>
```

Our person in charge said we needed a prefix

### Expected Behavior

```
 <Link href="/about">
          <a>About Us</a>
        </Link>
```

 There is no splicing prefix for a label after packaging ， 

Online environment   Should be such an address
```
<a class="" target="_blank" rel="noreferrer" href="https://pctest.vrtbbs.com/about">About Us</a>

But for now:  <a class="" target="_blank" rel="noreferrer" href="/about">About Us</a>
```

Our person in charge said we needed a prefix

### To Reproduce

```
 <Link href="/about">
          <a>About Us</a>
        </Link>
```

 There is no splicing prefix for a label after packaging ， 

Online environment   Should be such an address
```
<a class="" target="_blank" rel="noreferrer" href="https://pctest.vrtbbs.com/about">About Us</a>

But for now:  <a class="" target="_blank" rel="noreferrer" href="/about">About Us</a>
```

Our person in charge said we needed a prefix

**Comment by parlay96** [comment]
The person in charge of SEO said that our links need prefixes

**Comment by wwwdepot** [comment]
Use a custom component. See https://nextjs.org/docs/api-reference/next/router#userouter
